### ISSUE TITLE
Speed drop

### ISSUE THREAD

**Issue by gedeminas** [ferares/gulp-remove-sourcemaps#4]
After updating from 1.0.1 to 1.0.2, speed in decreased very dramatically.
I created a project for testing the problem, including two popular js libs which I regularly use in my projects:
```
const
    gulp = require('gulp'),
    concat = require('gulp-concat'),
    removeSourcemaps = require('gulp-remove-sourcemaps');

function js() {
    return gulp.src([
            './node_modules/bootstrap/dist/js/bootstrap.bundle.min.js',
            './node_modules/swiper/swiper-bundle.min.js',
        ])
        .pipe( removeSourcemaps() )
        .pipe( concat('main.js') )
        .pipe( gulp.dest('./scripts') )
        .pipe( gulp.dest('./scripts') );
}

exports.default = js;
```
Difference between runing two versions is huge. 1.0.1:
```
[10:45:15] Using gulpfile ~/Downloads/test/gulpfile.js
[10:45:15] Starting 'default'...
[10:45:16] Finished 'default' after 242 ms
```
After updating to 1.0.2:
```
[10:43:54] Using gulpfile ~/Downloads/test/gulpfile.js
[10:43:54] Starting 'default'...
[10:44:39] Finished 'default' after 45 s
```

**Comment by ferares** [comment]
@gedeminas interesting, maybe I can improve the regex

**Comment by ferares** [comment]
Ok, fixed on version 1.0.4
